common/eth2_network_config/built_in_network_configs/gnosis/config.yaml

```
@@ -89,4 +89,15 @@ DEPOSIT_CONTRACT_ADDRESS: 0x0B98057eA310F4d31F2a452B414647007d1645d9
 
 # Network
 # ---------------------------------------------------------------
-SUBNETS_PER_NODE: 4
\ No newline at end of file
+SUBNETS_PER_NODE: 4
+GOSSIP_MAX_SIZE: 10485760
+MAX_REQUEST_BLOCKS: 1024
+MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
+MAX_CHUNK_SIZE: 1048576
```

```suggestion
MAX_CHUNK_SIZE: 10485760
```
From https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L116